src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -38,6 +38,6 @@ def test_subtest(self):
 
         Use Cases: core tests for daos_test
 
-        :avocado: tags=all,regression,vm,pr,hw,daos_test
+        :avocado: tags=all,full_regression,pr,daos_test
```

Comment:
Nak.  _regression_ is the current tag for PRs.  _full_regression_ is the weekly test run tag.  So just remove _full_regression_ to fix this.